js/components/dataset/resource/file-form.vue

```diff
@@ -1,13 +1,19 @@
 <style lang="less">
+@import '~less/admin/variables';
+
 .resource-upload-dropzone {
-    border: 4px dashed #bbb;
+    border: 4px dashed @gray-lte;
```

Comment:
@noirbizarre I can see you hooked on the _variables_ thingy 🙂

Reply:
Yes, I always had, but there was no proper `@import ~` support in vue-loader initially so it was not possible to properly use them.
When I encounter a case were I can use them, I try to do/fix it